corehq/apps/sms/tests/test_registration.py

```diff
@@ -27,8 +25,9 @@ class RegistrationTestCase(BaseSMSTest):
 
     def setUp(self):
         super(RegistrationTestCase, self).setUp()
+        delete_all_users()
 
-        self.domain = 'sms-reg-test-domain'
+        self.domain = 'sms-reg-test-domain-1'
```

Comment:
why was this needed? and also the "delete_all_users" in beginning. Are these left overs while writing tests?

Reply:
Whoops, yes I left this in by accident.